plugins/storage/volume/linstor/src/main/java/org/apache/cloudstack/storage/datastore/util/LinstorUtil.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.cloudstack.storage.datastore.util;
```

Comment:
@rp- general remark - you want to add the Apache License headers to all new files.